node/container.go

```diff
@@ -65,6 +87,7 @@ func (a *HostAgent) StopContainer(serviceID string, instanceID int) error {
 		return err
 	}
 
+	a.setInstanceState(serviceID, instanceID, zkservice.StateStopped)
```

Comment:
Remove this line, this will be handled by the event monitor
https://github.com/control-center/serviced/pull/3352/files#diff-fbf6fa650618db48c5e4bcb7ad4b84d4R410